docs/source/index.md

```diff
@@ -14,3 +14,8 @@ Here is what I think happens when you turn the light on while acquiring an image
 :caption: Sections
 
 ```
+
+## Tom
```

Comment:
Hey, I think your text should come before the `{toctree}` section. Could you please move it above?